endpoint.go

```diff
@@ -417,36 +297,30 @@ func (ep *endpoint) Join(containerID string, options ...EndpointOption) error {
 		}
 	}()
 
-	err = ep.buildHostsFiles()
+	err = sb.buildHostsFiles(ep)
```

Comment:
What I meant was this should completely become an option passed while creating a sandbox, isn't it? Because this should happen once during sandbox create and there should be no way to add them for every endpoint.